.releaserc.json

```diff
@@ -4,7 +4,7 @@
   ],
   "plugins": [
     ["@semantic-release/github", {
-      "assets": ["go-earlybird-linux", "go-earlybird-macos", "go-earlybird.exe"],
+      "assets": ["go-earlybird-linux", "go-earlybird-macos", "go-earlybird.exe", "go-earlybird-arm64-macos"],
```

Comment:
I'm not sure if we want to build fat, or just arm? My personal preference would be fat, but the trend on the internet appears to be platform specific, so as long as we can still build on Intel and Apple silicon at the same time, I guess this is fine.

Reply:
go build natively does not support fat binaries. We would need to use something which is not part of go-build -> https://github.com/golang/go/issues/53281